### PR TITLE
[Feature] - Added Typography component in the library

### PR DESCRIPTION
**Pull request by parthpandyappp**
### Directory structure and file justification
Added a new directory named alerts under which markup has been written in `typography.html` and the stylesheet file is written in `typography.css`. The stylesheet file `typography.css` only consists of the styles related to buttons i.e it is a feature-specific stylesheet file that is later getting integrated with the main stylesheet file in the root directory i.e `styles.css`

### Explicit integration of stylesheet file
As we see, many open-source CSS libraries generally have a CDN link that is embedded and linked to the head section of any markup file. Similarly, a single main styles.css file is responsible to provide access to all the other stylesheets.

## About this PR
This PR adds a Badge component of different types.

### CODE REVIEW

File: typography/typography.css [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  text-align: left;
+  margin: 1rem;
+}
+
+.sub-title {
+  font-size: 1.2rem;
+}
+
+.h-1 {
+  font-size: 48px;
```
Comment (sonishreyas) [inline typography/typography.css]: you can change these to rems so it will also change according to screens.

**Review by sonishreyas — COMMENTED** [review]
The rest looks good. You can change those to rems if you want, I just think that it will help you in making the site responsive.

File: typography/typography.css (hunk 1 above)
Comment (parthpandyappp) [reply typography/typography.css]: Done, man!

File: typography/typography.css [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  font-weight: 400;
+}
+
+.sm-bold {
+  font-weight: 600;
+}
+.bold {
+  font-weight: 700;
+}
+.black {
```
Comment (TejasShekar) [inline typography/typography.css:63]: extra-bold would be a better class name than black IG

File: typography/typography.html [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+          </div>
+          <li class="sub-title">Texts</li>
+          <div class="disp-typo">
+            <p class="h-1">Lorem Ipsum</p>
+            <p class="h-2">Lorem Ipsum</p>
+            <p class="h-3">Lorem Ipsum</p>
+            <p class="h-4">Lorem Ipsum</p>
+            <p class="h-5">Lorem Ipsum</p>
+          </div>
+          <li class="sub-title">Texts</li>
```
Comment (TejasShekar) [inline typography/typography.html]: "Text weights" would be a better title for this as the previous section also has the title "Texts".
Also, do mention the weight numbers on the side or below each of them.

**Review by TejasShekar — COMMENTED** [review]
Looks good but do check out my comments.

File: typography/typography.css (hunk 2 above)
Comment (parthpandyappp) [reply typography/typography.css:63]: Yaa, I'll update it. Thanks for your valuable suggestions.

File: typography/typography.html (hunk 3 above)
Comment (parthpandyappp) [reply typography/typography.html]: Cool, It'll be also done. Thanks for being up here :+1:

**Comment by parthpandyappp** [comment]
Fixed it!

**Pull request merged**